src/enc_jpeg2000.c

```diff
@@ -73,7 +73,10 @@ int SUB_NAME(unsigned char *cin,g2int *pwidth,g2int *pheight,g2int *pnbits,
     jas_image_t image;
     jas_stream_t *jpcstream,*istream;
     jas_image_cmpt_t cmpt,*pcmpt;
-#define MAXOPTSSIZE 1024
+/**
+ * \def MAXOPTSSIZE
+*/
+#define MAXOPTSSIZE 1024 
```

Comment:
Try it like this:

`#define MAXOPTSSIZE 1024 /**< Maximum size of the options. */`

Reply:
@edwardhartnett 
I have try this before. it does not work for the warning.

Reply:
This was said as a (macro definition), which was discussed a lot in the doxygen community. But not a suggested way(1) use \def or (2) use auto link way works for this code.

Reply:
I am using this in g2c and it is working. But now that I think about it, I had trouble with F90 code in PIO and doxygen.

For now, I suggest you ignore this warning and press on with the rest of the code. We will circle around again when the code is mostly doxygenated and sort out the remaining warnings....